web-components/src/types/rdf/batch-metadata.ts

```diff
@@ -0,0 +1,17 @@
+import { NameUrl } from './';
```

Comment:
what about renaming this file to something more consistent with what's done in regen-registry-standards, eg `C01-verified-carbon-standard-batch.ts`

Reply:
sounds good